orchagent/aclorch.cpp

```diff
@@ -3974,6 +4005,19 @@ bool AclOrch::addAclTable(AclTable &newTable)
             m_mirrorV6TableId[table_stage] = table_id;
         }
 
+        // We use SAI_ACL_BIND_POINT_TYPE_SWITCH for PFCWD table in DNX platform.
+        // This bind type requires to bind the table to switch.
+        string platform = getenv("platform") ? getenv("platform") : "";
+        string sub_platform = getenv("sub_platform") ? getenv("sub_platform") : "";
+        if (platform == BRCM_PLATFORM_SUBSTRING && sub_platform == BRCM_DNX_PLATFORM_SUBSTRING &&
+            newTable.type.getName() == TABLE_TYPE_PFCWD)
+        {
+            if(!bindEgrAclTableToSwitch(newTable))
```

Comment:
This function is called after `createBindAclTable`. If the table has to be bound to switch, will that cause problem?

Reply:
We bind egress acl table to switch only for PFCWD table on DNX platform. This is fine because, on DNX,  PFCWD table does not bind port anymore. So `createBindAclTable` will only create PFCWD table on DNX, without binding the table to any port.